test/encryption/encryption.test.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+const assert = require('assert');
+const mdb = require('mongodb');
+
+describe('environmental variables', () => {
+  it('MONGODB_TEST_URI is set', async function() {
+    const uri = process.env.MONGOOSE_TEST_URI;
+    assert.ok(uri);
+  });
+
+  it('CRYPT_SHARED_LIB_PATH is set', async function() {
+    const shared_library_path = process.env.CRYPT_SHARED_LIB_PATH;
+    assert.ok(shared_library_path);
+  });
+});
+
+describe('basic integration', () => {
+  it('supports mongodb csfle auto-encryption integration', async() => {
+    // 1. Create a MongoClient configured with auto encryption (referred to as `client_encrypted`)
+    const client = new mdb.MongoClient(
+      process.env.MONGOOSE_TEST_URI,
+      {
+        autoEncryption: {
+          keyVaultNamespace: 'keyvault.datakeys',
```

Comment:
Without a schemaMap (or an encryptedFieldsMap), we won't actually encrypt anything.  We need to provide a schema map or encrypted fields map that specifies that `encrypted` should be encrypted - otherwise we're just inserting a document into the cluster.

Reply:
Changed it to use a schema map, and checked that binary is inserted in!